x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -326,14 +326,20 @@ const ActivityItem: React.FunctionComponent<{ action: ActionStatus }> = ({ actio
     <EuiText>
       <FormattedMessage
         id="xpack.fleet.agentActivity.completedTitle"
-        defaultMessage="{nbAgents} {agents} {completedText}"
+        defaultMessage="{nbAgents} {agents} {completedText}{offlineText}"
         values={{
           nbAgents:
             action.nbAgentsAck === action.nbAgentsActioned
               ? action.nbAgentsAck
               : action.nbAgentsAck + ' of ' + action.nbAgentsActioned,
           agents: action.nbAgentsActioned === 1 ? 'agent' : 'agents',
           completedText: getAction(action.type).completedText,
+          offlineText:
+            action.status === 'ROLLOUT_PASSED' && action.nbAgentsActioned - action.nbAgentsAck > 0
+              ? `, ${
+                  action.nbAgentsActioned - action.nbAgentsAck
+                } agent(s) offline during the rollout period`
+              : '',
```

Comment:
Does this need a translation? If so it might be easier to pull it out into a variable to get the logic out of the JSX here.

Reply:
this is already part of a `FormattedMessage`, though it can't really be translated as the text consists of all variables:  `defaultMessage="{nbAgents} {agents} {completedText}{offlineText}"`